### ISSUE TITLE
v3.1 Quiz 3 Question #5 Hurts My Head

### ISSUE THREAD

**Issue by ZebGirouard** [GA-WDI/fundamentals#451]
Literally, the only difference between a right and wrong four-sentence answer with code syntax embedded is "a" vs "the" in one place.  Can we please please please not do this to our incoming students?  We can change one significantly, like say an `<a>` is for a heading or something, and we would still get the point across.

**Comment by GA-MEB** [comment]
I agree that this question could definitely be simplified. Let's tackle this in v4.1
